diff -urN gstreamer-0.6.3.orig/configure.ac gstreamer-0.6.3/configure.ac
--- gstreamer-0.6.3.orig/configure.ac	Wed Aug 27 06:55:09 2003
+++ gstreamer-0.6.3/configure.ac	Wed Aug 27 06:55:31 2003
@@ -37,7 +37,6 @@
 AS="${CC}"
 
 AS_COMPILER_FLAG(-Wall,GST_ERROR="$GST_ERROR -Wall",)
-AS_COMPILER_FLAG(-Werror,GST_ERROR="$GST_ERROR -Werror",)
 
 
 dnl We disable static building for development, for time savings
