diff -urN gstreamer-0.8.1.orig/docs/gst/Makefile.am gstreamer-0.8.1/docs/gst/Makefile.am
--- gstreamer-0.8.1.orig/docs/gst/Makefile.am	2004-04-17 11:41:53.334219104 +0200
+++ gstreamer-0.8.1/docs/gst/Makefile.am	2004-04-17 11:42:27.809977992 +0200
@@ -42,7 +42,7 @@
 SCAN_OPTIONS=
 
 # Extra options to supply to gtkdoc-mkdb.
-MKDB_OPTIONS=--sgml-mode --ignore-files=trio
+MKDB_OPTIONS=--sgml-mode --ignore-files=parse
 
 # Extra options to supply to gtkdoc-fixref.
 FIXXREF_OPTIONS=
