diff -Nuard gstreamer-0.6.4.orig/common/m4/gst-doc.m4 gstreamer-0.6.4/common/m4/gst-doc.m4
--- gstreamer-0.6.4.orig/common/m4/gst-doc.m4	2003-10-10 11:27:14.000000000 +0200
+++ gstreamer-0.6.4/common/m4/gst-doc.m4	2003-10-16 22:31:32.000000000 +0200
@@ -149,7 +149,7 @@
 AM_CONDITIONAL(BUILD_DOCS,          test "x$BUILD_DOCS" = "xyes")
 AM_CONDITIONAL(BUILD_PLUGIN_DOCS,   test "x$BUILD_PLUGIN_DOCS" = "xyes")
 AM_CONDITIONAL(DOC_HTML,            $DOC_HTML)
-AM_CONDITIONAL(DOC_PDF,             $DOC_PDF)
-AM_CONDITIONAL(DOC_PS,              $DOC_PS)
+AM_CONDITIONAL(DOC_PDF,             false)
+AM_CONDITIONAL(DOC_PS,              false)
 ])
 
